acme/db/nosql/challenge.go

```diff
@@ -6,8 +6,10 @@ import (
 	"time"
 
 	"github.com/pkg/errors"
-	"github.com/smallstep/certificates/acme"
+
 	"github.com/smallstep/nosql"
```

Comment:
why are we splitting these up one per line like this?

Reply:
The split is for imports from the stdlib, third parties, 1st party and local imports. So far I've been doing it when I change a file with imports. I'd like to automate it sometime using https://github.com/incu6us/goimports-reviser (using their `company` options)